src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -79,3 +79,27 @@
     type: command
     short-summary: Wait for a Kubernetes cluster to reach a desired state.
 """
+helps['aks install-connector'] = """
+    type: command
+    short-summary: Deploy the ACI-Connector to a AKS cluster.
+    examples:
+        - name: Install the ACI-Connector to an AKS Cluster.
+          text: az aks install-connector --name MyManagedCluster --resource-group MyResourceGroup --connector_name MyConnector
```

Comment:
The argument name should be `--connector-name` not ``--connector_name`.
Same elsewhere in this help file.